.github/CONTRIBUTING.md

```diff
@@ -43,18 +43,36 @@ You can (and should) run our test suite using [*tox*].
 However, you’ll probably want a more traditional environment as well.
 We highly recommend to develop using the latest Python release because we try to take advantage of modern features whenever possible.
 
-Clone the *attrs* repository to your computer:
+First [fork](https://github.com/python-attrs/attrs/fork) the repository on GitHub.
+
+Clone the fork to your computer:
 
 ```console
-$ git clone git@github.com:python-attrs/attrs.git
+$ git clone git@github.com:<your-username>/attrs.git
 ```
 
 Or if you prefer to use Git via HTTPS:
 
 ```console
-$ git clone https://github.com/python-attrs/attrs.git
+$ git clone https://github.com/<your-username>/attrs.git
+```
+
+Then add the *attrs* repository as *upstream* remote:
+
+```console
+$ git remote add upstream git@github.com:python-attrs/attrs.git
+```
+
+You can of course also rename the *origin* remote (your fork) to *fork* and add the repository as *origin*.
```

Comment:
This gives people many options but if they are not proficient with Git, it'll probably only confuse them. OTOH, if they do know Git well, they can come up with this idea on their own. Besides, there's nothing in this guide referring to a remote called `fork` so I'd probably just drop this hint.